e2e/suites/user.test.ts

```diff
@@ -0,0 +1,98 @@
+import insertFixtures from '../helpers/insertFixtures';
+import proxyMock from '../helpers/proxyMock';
+import { adminLogin, login, logout } from '../helpers/login';
+import { host } from '../config';
+
+describe('User', () => {
+  beforeAll(async () => {
+    await insertFixtures();
+    await proxyMock();
+    await adminLogin();
+    await page.goto(`${host}/settings/users`);
+  });
+
+  afterAll(async () => {
+    await logout();
+  });
+
+  const getListRows = async () =>
+    page.$$eval('tbody tr', rows =>
+      rows.map(row => {
+        const tds = [...row.getElementsByTagName('td')];
+        return tds.map(td => td.textContent);
+      })
+    );
+
+  it('Should list all the users', async () => {
+    await page.waitForSelector('tbody tr:nth-child(1)');
+    const userRows = await getListRows();
+    expect(userRows[0]).toEqual(['admin', 'Password', 'admin', ' Activistas']);
+    expect(userRows[1]).toEqual(['editor', 'Password', 'editor', ' Activistas Asesores legales']);
+  });
+
+  describe('Editing user', () => {
+    const assertInputValue = async (selector: string, expected: string) => {
+      const fieldValue = await page.$$eval(
+        selector,
+        input => (<HTMLInputElement | HTMLSelectElement>input[0]).value
+      );
+      expect(fieldValue).toEqual(expected);
+    };
+
+    it('Should open the data of the user in a side panel', async () => {
+      await expect(page).toClick('td', { text: 'admin' });
+      await assertInputValue('#email_field > input', 'admin@uwazi.com');
+      await assertInputValue('#role_field > select', 'admin');
+      await assertInputValue('#username_field > input', 'admin');
+
+      const groups = await page.$$eval('.multiselectItem', items =>
+        items.map(item => ({
+          checked: (<HTMLInputElement>item.children[0]).checked,
+          username: item.textContent,
+        }))
+      );
+      expect(groups[0].username).toEqual('Activistas');
+      expect(groups[0].checked).toBe(true);
+      expect(groups[1].username).toEqual('Asesores legales');
+      expect(groups[1].checked).toBe(false);
```

Comment:
We probably need to check that the count of checked groups is 2 to make this complete, isn't it?

Reply:
Right, done.